gammapy/utils/units.py

```diff
@@ -0,0 +1,44 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Units and Quantity related helper functions"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+import astropy.units as u
+
+__all__ = [
+    'standardise_unit',
+]
+
+
+def standardise_unit(unit):
+    """Standardise unit.
+    
+    Changes applied by this function:
+
+    * Drop "photon" == "ph" from the unit
+    * Drop "count" == "ct" from the unit
+
+    Parameters
+    ----------
+    unit : `~astropy.units.Unit` or str
+        Any old unit
+
+    Returns
+    -------
+    unit : `~astropy.units.Unit`
+        Shiny new, standardised unit
+
+    Examples
+    --------
+    >>> from gammapy.utils.units import standardise_unit
+    >>> standardise_unit('ph cm-2 s-1')
+    Unit("1 / (cm2 s)")
+    >>> standardise_unit('ct cm-2 s-1')
+    Unit("1 / (cm2 s)")
+    >>> standardise_unit('cm-2 s-1')
+    Unit("1 / (cm2 s)")
+    """
+    unit = u.Unit(unit)
+    for base, power in zip(unit.bases, unit.powers):
```

Comment:
If you go this route, I would actually create new bases and powers and just make a new `CompositeUnit` with those (with the same scale). Though what you have of course essentially does the same thing under the hood.